examples/opt_serving/model/wrapper.py

```diff
@@ -184,6 +191,30 @@ def grouped_reorder_cache(arys, device_mesh):
             for layer_loc in self.cache_location)
 
 
+class AlpaInferenceFunc(WrappedInferenceFunc):
+
+    def __init__(self, *args, max_target_positions):
+        super().__init__(*args)
+        self._max_target_positions = max_target_positions
+
+    def _process_attention_mask(self, attention_mask):
+        if isinstance(attention_mask, torch.Tensor):
+            attention_mask = attention_mask.cpu().numpy()
+
+        # attention_mask =  attention_mask == 1
+        batch_size = attention_mask.shape[0]
+        # # flip
+        # big_neg = jnp.finfo(dtype).min
+        # flipped = (attention_mask == 0) * big_neg
+        # # extend
+        ret_mask = np.ones((batch_size, self._max_target_positions),
```

Comment:
one or zero?

Reply:
does not matter because positions that are autoregressive generated won't be impacted by masks.